sdk/resourcemanager/cosmos/armcosmos/CHANGELOG.md

```diff
@@ -1,5 +1,114 @@
 # Release History
 
+## 3.0.0-beta.5 (2024-06-21)
+### Breaking Changes
+
+- Type of `DataTransferJobProperties.Error` has been changed from `*ErrorResponse` to `*ErrorResponseAutoGenerated`
```

Comment:
duplicate schema?

Reply:
`modelerfour.lenient-model-deduplication: true` is added [here](https://github.com/Azure/azure-rest-api-specs/pull/23327). It is currently in preview and we cannot check it. But I see that there is no duplicate schema in the [latest stable version](https://github.com/Azure/azure-rest-api-specs/pull/28367) of cosmos.